pkg/db/sql_api.go

```diff
@@ -30,7 +48,9 @@ func WriteMsg(db *sql.DB, msg msgs.CAN) error {
 func TimeWindowedRead(db *sql.DB, canID uint16, from time.Time, to time.Time) ([]msgs.CAN, error) {
 	rows, err := db.Query(`
     SELECT
+      can.source,
       can.id,
+      can.rtr,
```

Comment:
We don't actually use the RTR bit anywhere. It would be more useful to store the DLC?

Reply:
Done. I'll just store both.